rust/src/detect/byte_math.rs

```diff
@@ -19,6 +19,7 @@
 
 use crate::detect::error::RuleParseError;
 use crate::detect::parser::{parse_token, take_until_whitespace};
+use crate::detect::*;
```

Comment:
Is it not better to specify the precise list than having a wildcard ?

Reply:
Good point; I'll make an update.